conf/defaults.ini

```diff
@@ -931,6 +931,10 @@ max_annotations_to_keep =
 # Configures the batch size for the annotation clean-up job. This setting is used for dashboard, API, and alert annotations.
 cleanupjob_batchsize = 100
 
+# Enforces the maximum allowed length of the tags for any newly introduced annotations. It can be between 1 and 4096 inclusive (which is the respective's column length). Default value is 500.
+# Setting it to a higher value would impact performance therefore is not recommended.
+tags_length = 500
```

Comment:
QQ: Is there a performance problem with more than 500? Could we skip this conf?

Reply:
I don't know; I got the impression that you had suggested it 😳